admin_guide/preventing_node_placement.adoc

```diff
@@ -0,0 +1,101 @@
+= Preventing Specific Node Placement
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can prevent users from targeting specific nodes
+when scheduling pods using labels and node selectors, as well as the node
+blacklist setting.  
+
+== Constraining Pod Placement
+
```

Comment:
Maybe add a bit more detailed introduction here:

"Enabling this plugin will require the `pods/binding` permission for roles which need to target particular nodes via the `nodeName` Pod spec field. It will also enforce constraints on the use of the `nodeSelector` Pod spec field based on the `pods/binding` permission and the `nodeSelectorLabelBlacklist` configuration option."